### ISSUE TITLE
Blocking I/O operation

### ISSUE THREAD

**Issue by SlimeDog** [PyroTempus/My-Plugins#45]
Spigot 1.15.1 (Jenkins 2558)
PyroWeatherPro 3.5.2

On player join (in a world not configured for PyroWeather)
```
[08:07:09] [Server thread/WARN]: [LagMonitor] Plugin PyroWeatherPro is performing a blocking I/O operation (Socket: https://api.spigotmc.org/legacy/update.php?resource=60394) on the main thread. This could affect the server performance, because the thread pauses until it gets the response. Such operations should be performed asynchronous from the main thread. If this happens on server startup, fixing it can drastically reduce startup time. Keep in mind to keep the code thread-safe. Report it to the plugin author
[08:07:09] [Server thread/WARN]: [LagMonitor] Source: VersionUpdater.java, method checkForUpdates, line 47
```

**Comment by PyroTempus** [comment]
I don't use LagMonitor - however - method moved to async thread which should resolve the issue, please make a ticket on discord or DM me if the issue persists on v3.5.3.

**Comment by SlimeDog** [comment]
I didn't expect that you use LagMonitor, but I wanted to be clear where the report was made.

Confirmed fixed in PWP 3.5.3
